docs/sql-data-sources-jdbc.md

```diff
@@ -275,11 +275,22 @@ logging into the data sources.
     </td>
     <td>read/write</td>
   </tr>  
+
+  <tr>
+   <td><code>connectionProvider</code></td>
```

Comment:
I think this option conflicts with `spark.sql.sources.disabledJdbcConnProviderList` (see also https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/jdbc/README.md#how-to-disable-jdbc-connection-providers). This option is an allow list and that specific option is a disallow list.

Reply:
Oh, okay. I just noticed that you mentioned `spark.sql.sources.disabledJdbcConnProviderList` in the Pr description.